config.dsc

```diff
@@ -444,7 +443,9 @@ config({
                 // Extra dependencies to make MSBuild work
                 { id: "Microsoft.VisualStudio.Setup.Configuration.Interop", version: "1.16.30"},
                 { id: "System.CodeDom", version: "4.4.0"},
-                { id: "Microsoft.Build.Locator", version: "1.0.31"},
+
+                // Used for MSBuild input/output prediction
+                { id: "Microsoft.Build.Prediction", version: "0.1.0" },
 

```

Comment:
Removing at this package is unused. See `MsBuildAssemblyLoader.cs` for how we load MSBuild manually.